engine/lib/views.php

```diff
@@ -739,6 +739,8 @@ function elgg_view_layout($layout_name, $vars = array()) {
  * @param array                      $vars An associative array of display options for the menu.
  *
  *                          Options include:
+ *                              items => an array of unprepared menu items
+ *                                       as ElggMenuItem or menu item factory options
```

Comment:
If folks want to avoid existing hook interference they can choose a unique menu name.